v2/internal/controllers/crd_networking_privateendpoint_test.go

```diff
@@ -0,0 +1,117 @@
+/*
+Copyright (c) Microsoft Corporation.
+Licensed under the MIT license.
+*/
+
+package controllers_test
+
+import (
+	"testing"
+
+	"github.com/Azure/go-autorest/autorest/to"
+	. "github.com/onsi/gomega"
+
+	"github.com/Azure/azure-service-operator/v2/api/network/v1beta20201101"
+	network "github.com/Azure/azure-service-operator/v2/api/network/v1beta20220701"
+	resources "github.com/Azure/azure-service-operator/v2/api/resources/v1beta20200601"
+	storage "github.com/Azure/azure-service-operator/v2/api/storage/v1beta20210401"
+	"github.com/Azure/azure-service-operator/v2/internal/testcommon"
+)
+
+func Test_Networking_PrivateEndpoint_CRUD(t *testing.T) {
+	t.Parallel()
+
+	tc := globalTestContext.ForTest(t)
+
+	rg := tc.CreateTestResourceGroupAndWait()
+
+	saKind := storage.StorageAccount_Kind_Spec_BlobStorage
+	sa := newStorageAccount(tc, rg)
+	sa.Spec.Kind = &saKind
+
+	vnet := newVMVirtualNetwork(tc, testcommon.AsOwner(rg))
+	subnet := newVMSubnet(tc, testcommon.AsOwner(vnet))
+
+	tc.CreateResourcesAndWait(sa, vnet, subnet)
+
+	tc.Expect(sa.Status.Id).ToNot(BeNil())
+
+	endpoint := newPrivateEndpoint(tc, rg, sa, subnet)
```

Comment:
Minor: Same question as in other test. Can we make this in parallel too? That's how it would be done w/ `kubectl apply`

So:
```
tc.CreateResourcesAndWait(sa, vnet, subnet, endpoint)
```